src/libraries/Microsoft.Extensions.Logging/gen/LoggerMessageGenerator.Parser.cs

```diff
@@ -427,88 +415,32 @@ public IReadOnlyList<LoggerClass> GetLogClasses(IEnumerable<ClassDeclarationSynt
                 return (loggerField, false);
             }
 
-            private (int eventId, int? level, string? message, string? eventName) ExtractAttributeValues(AttributeArgumentListSyntax args, SemanticModel sm)
+            private (int eventId, int? level, string message, string? eventName) ExtractAttributeValues(AttributeArgumentListSyntax args, SemanticModel sm)
```

Comment:
Removing dead code here and simplifying the method logic. 

The attribute values for LoggerMessageAttribute can only be property assignments

(can only be in the format `LogLevel =`,  will NOT be either positional: `, LogLevel.Debug` or specifiable with colon: `level: LogLevel.Debug`)